constellations/gemini-user/src/announcements/announce.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+require("./announce.sass");
+import React from "react";
+import API from "../utils/api.js";
+import { keys, sortBy, groupBy } from "lodash";
+import moment from "moment";
+
+export class AnnouncementGroup extends React.Component {
+    render() {
+        let a = this.props.announcements;
+        var postedAt = moment(this.props.postedAt).format('l');
```

Comment:
In new Javascript, `let` and `const` are superior to `var`. In javascript, `var` has some extra behavior with global scoping that can often confuse people. `let` and `const` were created to be more like other programming languages when it comes to variable scoping.

Since postedAt never changes, you make it a `const`.